src/test/java/org/jabref/model/entry/identifier/RFCTest.java

```diff
@@ -0,0 +1,39 @@
+package org.jabref.model.entry.identifier;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.net.URI;
+import java.util.Optional;
+
+import org.junit.jupiter.api.Test;
+
+class RFCTest {
+
+    @Test
+    void testParsePlainRfcId() {
```

Comment:
Convert to `@ParameterizedTest`.You can use `@CsvSource`. Left: `excpected`, right: input.

Use `assertEquals(expected, new RFC(input).getNormlized()`. 

Also the next test case.

Reply:
Converted `testParsePlainRfcId` to a `@ParameterizedTest` using `@CsvSource`(done)
- Updated `testGetExternalUri` to verify the URI using `Optional.of()` for consistent checks.